docs/customization/extensions.rst

```diff
@@ -0,0 +1,75 @@
+Extensions
+==========
+Saleor has implemented extensions architecture.
+It includes hooks for most basic operations like calculation of prices in the checkout or
+calling some actions when an order has been created.
+
+
+ExtensionsManager
```

Comment:
I think we should start with "how to use it" before we dive into the details of implementing a custom plugin and eventually a custom manager (not sure if we should event document that part).

Reply:
I think https://docs.djangoproject.com/en/2.2/topics/http/middleware/ is a good reference for what type of information we would need to provide (but I would still document installing plugins before writing custom ones).

Reply:
@patrys I've changed the structure as you said. Can you take a look one more time?